app/views/coronavirus_form/know_nhs_number.html.erb

```diff
@@ -8,14 +8,15 @@
     <% end %>
     
     <%= form_tag({},
-      "data-module": "track-coronavirus-form-vulnerable-people-know_nhs_number",
+      "data-module": "track-coronavirus-form-vulnerable-people-know-nhs-number",
```

Comment:
For tomorrow: I don't think this is needed, it's got underscores in all the other places...

Reply:
You're right this isn't needed 😄